packages/assistify-help-request/client/views/creationDialog/AssistifyCreateRequest.js

```diff
@@ -162,6 +167,13 @@ Template.AssistifyCreateRequest.onCreated(function() {
 	instance.requestTitle = new ReactiveVar('');
 	instance.openingQuestion = new ReactiveVar('');
 
+	if (FlowRouter._current.queryParams) {
+		const expertise = FlowRouter._current.queryParams['topic'] || FlowRouter._current.queryParams['expertise'];
```

Comment:
I assume that `FlowRouter` is the Rocket.Chat `deafult` for accessing request params.

Reply:
yup.